apps/dashboard/components/logs/queries/list-group.tsx

```
@@ -0,0 +1,216 @@
+
+  return (
+    <div className="w-full">
+      <div
+        className={cn(
+          "flex flex-row hover:bg-gray-2 cursor-pointer whitespace-nowrap rounded-[8px] pb-[9px] w-full",
+          index === selectedIndex ? "bg-gray-2" : "",
+        )}
+      >
+        <div
+          className={cn("flex flex-col w-11/12", `tabIndex-${index}`)}
+          role="button"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<button>
<input type="button">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)